sbt/src/main/scala/stryker4s/sbt/runner/SbtMutantRunner.scala

```diff
@@ -9,13 +9,11 @@ import sbt._
 import sbt.internal.LogManager
 import stryker4s.config.{Config, TestFilter}
 import stryker4s.extension.FileExtensions._
-import stryker4s.extension.exception.InitialTestRunFailedException
 import stryker4s.model._
 import stryker4s.mutants.findmutants.SourceCollector
 import stryker4s.report.Reporter
 import stryker4s.run.MutantRunner
 import stryker4s.sbt.Stryker4sMain.autoImport.stryker
-import sbt.Tests.Output
 
 class SbtMutantRunner(state: State, sourceCollector: SourceCollector, reporter: Reporter)(implicit config: Config)
```

Comment:
What is a `MutantRunner`? I'd imagine it is the `TestRunner` adapter for Stryker4s? Is it also used for the initial test run?

Reply:
Yes and yes. Does this mean I've named it correctly? I'm not sure TBH. It's responsible for running tests with the correct mutation activated (there's one for sbt, maven, command-runner)